clients/js/test/updateV2.test.ts

```diff
@@ -1046,7 +1048,91 @@ test('it can change an asset collection using same update authority (delegate ex
   });
 });
 
-test('it cannot remove an asset from collection using update delegate', async (t) => {
+test('it can remove an asset from collection as Collection authority with update delegate on asset', async (t) => {
```

Comment:
Interesting what's the signficance of having the update delegate on asset here?

Reply:
We're blocking changing the collection by an asset UpdateDelegate so here I'm explicitly checking and making sure I don't break normal operations. E.g. my logic doesn't always fail if an asset has an UpdateDelegate, it only fails when the Asset UpdateDelegate is the one removing the collection.

Reply:
Yes now I remember how this relates to the asset UpdateDelegate fix